fusor-ember-cli/app/controllers/subscriptions/management-application.js

```diff
@@ -14,23 +15,31 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
 
   showAlertMessage: false,
   showWaitingMessage: false,
+  showErrorMessage: false,
 
   msgWaiting: Ember.computed('newSatelliteName', function() {
     return ('Adding ' + this.get('newSatelliteName') + ' ....');
   }),
 
+  isValidMgmtAppName: Ember.computed('newSatelliteName', function(){
+    return ValidationUtil.validateMgmtAppName(this.get('newSatelliteName'));
```

Comment:
I get `ValidationUtil.default.validateMgmtAppName is not a function`

@dymurray, I assume that forgot to include changes to `validation-util.js` in the PR